vendor/brave-ios/Ledger/Data/Model/ContributionInfo.h

```diff
@@ -13,7 +13,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 + (NSFetchRequest<ContributionInfo *> *)fetchRequest;
 
-@property (nonatomic) int32_t category;
+@property (nonatomic) int32_t type;
```

Comment:
Changing the name of this attribute requires updating:
1. The CoreData entity
2. The `@dynamic category` declaration in ContributionInfo.m

Reply:
Fixed